dnf/cli/commands/shell.py

```diff
@@ -271,8 +270,10 @@ def _transaction(self, args=None):
         elif cmd == 'run':
             try:
                 self.base.do_transaction()
-            except:
-                pass
+            except Exception as e:
```

Comment:
What about to replace `except Exception as e`: by `except dnf.exceptions.Error as e:`?